### ISSUE TITLE
Emojis button

### ISSUE THREAD

**Issue by Plasmmer** [AnthyG/ThunderWave#39]

**Comment by AnthyG** [comment]
Just a button next to the message-input or what 🤔

**Comment by Plasmmer** [comment]
To access all emojis.

**Comment by AnthyG** [comment]
You may access all Emoji's on the official site by EmojiOne, [EmojiCodes](https://emoji.codes/).

**Comment by Plasmmer** [comment]
I mean for user access all emojis he can use!

**Comment by AnthyG** [comment]
The Emoji's on EmojiCodes are all Emoji's a user can use!

I can add this, but not now.

**Comment by Plasmmer** [comment]
But I'm talking about a button where user can intuitively click to show all emojis like Messenger/Skype/Hangouts and a lot chats does!

**Comment by AnthyG** [comment]
Yes!, I know, but I want to finish more important things, before I'm gone for 3 months..!

**Comment by AnthyG** [comment]
I had a lot of time to think about it, and I have to say, it's really needed!
But the main problem I realized, is that this would make the "Seed all Emoji's"-option in the Settings pretty much useless. Because, well, _all_ emoji's are shown in the list..

So I have to think on how I could implement it, so that not all of the emoji's are _loaded_ at once.
Maybe the button could just open up the same menu as when you type `:`, but in a static bar over or underneath the message-input!?

**Comment by Plasmmer** [comment]
It can use a tab menu, and only loads the emojis from the current tab and current view/page/scroll. After closing it, all these emojis are deleted, except the emojis the user has seen used by others or the emojis he used.

**Comment by AnthyG** [comment]
Why should it delete them again!?
That would be too dumb, but the tab-menu-idea is good!
Not sure though, how I can implement the scroll-part.. 🤔

**Comment by AnthyG** [comment]
I'm working on this for the ThunderWave Beta re-design, see #105 for more details.
